app/models/concerns/entity_search.rb

```diff
@@ -25,6 +68,35 @@ def self.search(query, opt = {})
       )
     end
 
+    # input: Entity | Integer, [ query<String> ], [ per_page<Integer> ]
```

Comment:
i think we should figure out a standard way we want to write these comment...perhaps [rdoc](https://github.com/ruby/rdoc)

Reply:
agreed! also, Flow makes sense for JS, but clearly is not a match for ruby code. in ruby, let's use a standard Ruby syntax and JS, use a standard JS one. and maybe note that in a style guide somewhere for contributors?

as for rdoc, it looks good. i dug into the syntax docs (which are a bit buried) and was eager to see how their syntax handles documenting complex types (eg: types composed of other types, like arrays, hashes, structs, etc...). in my experience these are the hardest things to document succinctly, and i rather like how Flow handles it (and find jsdoc to be a complete and utter failure in this regard, mainly because it stops at `Object` and leaves it at that.)

have you stumbled on that? could you link it here?